pkg/identityserver/client_registry.go

```diff
@@ -121,6 +121,7 @@ func (is *IdentityServer) createClient(ctx context.Context, req *ttnpb.CreateCli
 		req.Client.StateDescription = "admin approval required"
 		req.Client.SkipAuthorization = false
 		req.Client.Endorsed = false
+		req.Client.TieAccessToSession = false
```

Comment:
I don't think we need to force this to `false` for user-created clients _on create_.

But: your description in the API says that it can only be modified by admins, and `updateClient` doesn't prevent non-admins from updating the field.